### DISCUSSION TITLE
feature request: add simple mindmap extension

### DISCUSSION THREAD

**Discussion by viralpoetry** [standardnotes/forum#2492] [Suggestions]
Dear team,
It would be great to have simple mind mapping extension.
As an example I will put there link to a Text2MindMap github repo as it's simple yet effective tool

https://github.com/payne/text2mindmap

thank you for a great work,
Peter

**Comment by JaspalSuri** [comment]
Hi @viralpoetry, thank you for the suggestion. 🙂 I'm afraid that we are currently not planning on maintaining additional editors at this time (unless they serve as good replacements to our existing ones), but will consider this if we plan to offer a mindmapping extension in the future.

**Comment by imtoanle** [comment]
This's great extension too. I loved this idea.
Thanks @viralpoetry

**Comment by drhus** [comment]
mindmapping extension would be amazing
